rust/hyperlane-base/src/types/multisig.rs

```diff
@@ -1,3 +1,4 @@
+use itertools::Itertools;
```

Comment:
is this unused? remove pls

Reply:
no, it's used here: https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/b025a3af7e8cf4b96c0eaae18afdff2398235195/rust/hyperlane-base/src/types/multisig.rs#L179